libraries/CAN/library.properties

```diff
@@ -0,0 +1,10 @@
+name=CAN
+version=0.0.1
```

Comment:
Which is the right version here? Have we achieve completeness of the API features? then 1.0.0?
Are there still some feature missing? 0.1.0?

Reply:
Yes but only for XMC1400 😢 
I think we can start with 1.0.0

Reply:
yep, that is fine, if it is only for supporting some boards. 
The rest of the features are planned? Not planned for now?  @ederjc shall we document that somewhere?